<launch>

  <arg name="epochs" default="20"/>
  <arg name="steps_per_epoch" default="50"/>
  
  <arg name="folder" default="set1"/>
  <arg name="model" default="model1.h5"/>

  <node name="training_node" pkg="ackerbot" type="mltraining_lanefollow.py" output="screen">
    <param name="epochs" value="$(arg epochs)"/>
    <param name="steps_per_epoch" value="$(arg steps_per_epoch)"/>
    <param name="base_folder" value="$(arg folder)"/>
    <param name="modelname" value="$(arg model)"/>
  </node>
  
</launch>
